### PR TITLE
[CI:DOCS] Man pages: refactor common options: --pid

### PR DESCRIPTION
**Pull request by edsantiago**
I chose the one from podman-run, but reordered ns/private
to put them in alphabetical order.

Signed-off-by: Ed Santiago <santiago@redhat.com>
```release-note
more man-page deduplication
```

### CODE REVIEW

**Comment by edsantiago** [comment]
As with `--ipc`, I did not even try to combine with `podman-build` because it's too different: https://github.com/containers/podman/blob/34d516840df73787d2359037e4a2ab51849e59eb/docs/source/markdown/podman-build.1.md.in#L493-L500

**Comment by rhatdan** [comment]
LGTM
@containers/podman-maintainers PTAL

File: docs/source/markdown/options/pid.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#### **--pid**=*mode*
+
+Set the PID namespace mode for the container.
+The default is to create a private PID namespace for the container.
+
+- **container:**_id_: join another container's PID namespace;
+- **host**: use the host's PID namespace for the container. Note the host mode gives the container full access to local PID and is therefore considered insecure;
+- **ns:**_path_: join the specified PID namespace.
+- **private**: create a new namespace for the container (default)
```
Comment (lsm5) [inline docs/source/markdown/options/pid.md:9]: nit: semicolons after the first two, period after the third and nothing after the last.

**Review by lsm5 — APPROVED** [review]
LGTM with nit.

Ignore packit failures.Those will be addressed at https://github.com/containers/podman/pull/15457

File: docs/source/markdown/options/pid.md (hunk 1 above)
Comment (edsantiago) [reply docs/source/markdown/options/pid.md]: Oh, nice catch! Thank you! Fixed and pushed.

**Pull request merged**